tests/plugins/dialog/manual/ariaselected.html

```diff
@@ -0,0 +1,28 @@
+<div>
+	<textarea cols="80" id="editor" rows="10">
+		<p>Hello world!</p>
+	</textarea>
+
+	<button id="find-aria-selected">Find aria-selected tab</button>
+
+	<div>Aria value: <span id="aria-value"></span></div>
+
+	<script>
+		CKEDITOR.replace( 'editor' );
+
+		var button = CKEDITOR.document.findOne( '#find-aria-selected' ),
+			result = CKEDITOR.document.findOne( '#aria-value' ),
+			selectedTab;
+
+		button.on( 'click', function() {
+			selectedTab = CKEDITOR.document.findOne( '.cke_dialog_tab_selected' );
+			if ( !selectedTab ) {
+				result.setText( 'Open the dialog first!' );
+			} else {
+				result.setText( String( selectedTab.getAttribute( 'aria-selected' ) ) );
+			}
+
+			result.setStyle( 'color', result.getText() == 'true' ? 'green' : 'red' );
+		} );
```

Comment:
It's a neat idea to do this in such way, but a little confusing for me what's going on TBH (I mean when performing test). Maybe it could be done in a way that there is `Selected tab:` element (same as `Aria value:` now) and it is refreshed in real time so when one opens the dialog and changes tabs it updates (with selected tab name based on aria-attribute) so you can see if it works with live clicking. WDYT?

AFAIR we have an event for opening dialog, not sure about focusing tabs, so theoretically you can do it with simple `setInterval` an fetch `selectedTab` every x ms.